### DISCUSSION TITLE
I really just want to play target with my own personal music collection

### DISCUSSION THREAD

**Discussion by rasputozen** [ppy/osu#17784] [Ideas] [answered]
My ideal osu experience would be picking a song from my personal music collection, osu scans it to auto-generate bpm markers, and then I play target with my favorite music. I don't care about leaderboards at all beyond my own scores. I just want to listen to my music and click.

**Comment by peppy** [comment, marked as answer]
You are probably going to have to develop this yourself.

You can find other games that do this already if you're looking for an already made option (something like audiosurf). Results may vary.

**Reply by rasputozen** [reply]
Understandable. I tried to manually set the bpm marks of my songs in lazer recently but found the editor isn't operable yet. Whenever that's up I'll probably just do that. Thanks.
